test/IECoreScene/MeshAlgoFaceAreaTest.py

```diff
@@ -126,6 +128,37 @@ def testTwoFaces( self ) :
 		self.assertEqual( textureAreas[0], 6 )
 		self.assertEqual( textureAreas[1], 1 )
 
+	def testCancel( self ) :
+		canceller = IECore.Canceller()
+		cancelled = [False]
+
+		texture = False
```

Comment:
Might be a bit clearer to pass `texture` via `args` rather than have to declare it up here so it can be accessed in `backgroundRun()`?

Reply:
Addressed.